src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -295,12 +295,12 @@ protected override void ProcessRecord()
                                 else
                                 {
                                     // we write null out because:
-                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
+                                    // PS C:\> "$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}"
                                     // 3
                                     // so we also want
-                                    // PS C:\> �$null | ForEach-object aa | ForEach-Object {$_ + 3}�
+                                    // PS C:\> "$null | ForEach-object aa | ForEach-Object {$_ + 3}"
```

Comment:
Same comment as before: that was the intent of the original author, given the CP-1252-encoded curly quotes that I found. But I'll be happy to remove the quotes if they're actually incorrect.